services/registry/docker-compose.aws.yml

```diff
@@ -0,0 +1,10 @@
+version: '3.7'
+services:
+  registry:
+      image: registry:2
```

Comment:
FREEZE images: in this case ``registry:2.7.1`` https://hub.docker.com/_/registry